tests/interface-consumer/Makefile

```diff
@@ -1,10 +1,17 @@
+CHARMFILE := interface-consumer_amd64.charm
+build: $(CHARMFILE)
+
 lib/charms/microovn/v0/ovsdb.py: ../../microovn/lib/charms/microovn/v0/ovsdb.py
 	mkdir -p lib/charms/microovn/
 	cp -r ../../microovn/lib/charms/microovn/v0/ lib/charms/microovn/
 
-build: lib/charms/microovn/v0/ovsdb.py src/charm.py charmcraft.yaml
+lib/charms/tls_certificates_interface/v4/tls_certificates.py: ../../microovn/lib/charms/tls_certificates_interface/v4/tls_certificates.py
+	mkdir -p lib/charms/tls_certificates_interface
+	cp -r ../../microovn/lib/charms/tls_certificates_interface/v4/ lib/charms/tls_certificates_interface/
+
+$(CHARMFILE): lib/charms/microovn/v0/ovsdb.py lib/charms/tls_certificates_interface/v4/tls_certificates.py src/charm.py charmcraft.yaml
 	charmcraft pack
 
 clean:
-	rm -r lib/charms/ovsdb_interface/v0
+	rm -r lib/charms/microovn/v0
```

Comment:
Thanks for fixing the path! However, this fails if it does not exist, which calls for adding a `-f` argument to the `rm` commands.

We likely also want to remove the tls_certificates_interface copy on clean?